Cubical/Data/Int/Properties.agda

```diff
@@ -133,3 +100,156 @@ discreteInt (negsuc n) (negsuc m) with discreteℕ n m
 
 isSetInt : isSet Int
 isSetInt = Discrete→isSet discreteInt
+
+-- Compose sucPathInt with itself n times. Transporting along this
+-- will be addition, transporting with it backwards will be
+-- subtraction.
+
+addEq : ℕ → Int ≡ Int
+addEq zero = refl
+addEq (suc n) = compPath (addEq n) sucPathInt
+
+subEq : ℕ → Int ≡ Int
+subEq n i = addEq n (~ i)
+
+_+_ : Int → Int → Int
+m + pos n    = transport (addEq n) m
+m + negsuc n = transport (subEq (suc n)) m
+
+_-_ : Int → Int → Int
+m - pos zero    = m
+m - pos (suc n) = m + negsuc n
+m - negsuc n    = m + pos (suc n)
+
+-- We directly get that addition by a fixed number is an equivalence
+-- without having to do any induction!
+isEquivAddInt : (m : Int) → isEquiv (λ n → n + m)
+isEquivAddInt (pos n)    = isEquivTransport (addEq n)
+isEquivAddInt (negsuc n) = isEquivTransport (subEq (suc n))
```

Comment:
Why moving these definitions? It's quite inconvenient for reviewing.

Reply:
Because it was a different definition under different name (i.e. `_+'_` instead of `_+_`), then it was changed to replace the original one.

Reply:
Thanks.